docs/source/usrman/overview.rst

```diff
@@ -414,78 +392,73 @@ Environmental Management
 Initialization and Exit
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-Module functions :func:`MPI.Init` or :func:`MPI.Init_thread` and
-:func:`MPI.Finalize` provide MPI initialization and finalization
-respectively. Module functions :func:`MPI.Is_initialized()` and
-:func:`MPI.Is_finalized()` provide the respective tests for
+Module functions `Init` or `Init_thread` and `Finalize` provide MPI
+initialization and finalization respectively. Module functions
+`Is_initialized` and `Is_finalized` provide the respective tests for
 initialization and finalization.
 
 .. note::
 
-   :c:func:`MPI_Init()` or :c:func:`MPI_Init_thread()` is actually
-   called when you import the :mod:`MPI` module from the :mod:`mpi4py`
-   package, but only if MPI is not already initialized. In such case,
-   calling :func:`MPI.Init` or :func:`MPI.Init_thread` from Python is
-   expected to generate an MPI error, and in turn an exception will be
-   raised.
+   :c:func:`MPI_Init` or :c:func:`MPI_Init_thread` is actually called
```

Comment:
Nitpick: `:c:func:` functions are a bit confusing as they appear like other Python functions with unclickable names. How about making it simpler?
```suggestion
   ``MPI_Init()`` or ``MPI_Init_thread()`` is actually called
```

Reply:
Can we leave this issue for later? The proper fix should be implemented at the HTML CSS level.

Reply:
Sure!